packages/components/src/Heading/Heading.stories.tsx

```diff
@@ -9,48 +9,53 @@ export default {
 
 type Args = React.ComponentProps<typeof Heading>;
 
-const Template: Story<Args> = (args) => <Heading {...args} />;
-
-export const Heading1 = Template.bind(null);
-Heading1.args = {
-  size: "h1",
-  children: "Heading 1 24/32",
+export const Heading1: StoryObj<Args> = {
+  args: {
+    size: "h1",
+    children: "Heading 1 24/32",
+  },
```

Comment:
Notice that none of the stories actually define a `render` function...

For any story that does nothing but apply `args` to the component - i.e.

```tsx
(args) => <Heading {...args />;
```

we can omit it and Storybook will do that for us.